base_layer/wallet_ffi/wallet.h

```diff
@@ -326,33 +334,20 @@ struct TariPendingInboundTransaction *wallet_get_pending_inbound_transaction_by_
 bool wallet_test_complete_sent_transaction(struct TariWallet *wallet, struct TariPendingOutboundTransaction *tx,int* error_out);
 
 // Simulates the completion of a broadcasted TariPendingInboundTransaction
-bool wallet_test_transaction_broadcast(struct TariWallet *wallet, struct TariPendingInboundTransaction *tx,int* error_out);
+bool wallet_test_broadcast_transaction(struct TariWallet *wallet, ulonglong* tx_id, int* error_out);
+
+// Simulates receiving the finalized version of a TariPendingInboundTransaction
+bool wallet_test_finalize_received_transaction(struct TariWallet *wallet, ulonglong* tx_id, int* error_out);
 
 // Simulates a TariCompletedTransaction that has been mined
-bool wallet_test_mined(struct TariWallet *wallet, struct TariCompletedTransaction *tx,int* error_out);
+bool wallet_test_mine_transaction(struct TariWallet *wallet, ulonglong* tx_id, int* error_out);
```

Comment:
ulonglong should be unsigned long long, or unsigned long long int (they are equivalent).

Reply:
done